po/de.po

```diff
@@ -1,20 +1,21 @@
 #
 # Cvfosammmm <r.griesel@gmail.com>, 2020.
+# Stephan Lachnit <stephanlachnit@protonmail.com>, 2020.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: setzer\n"
-"Report-Msgid-Bugs-To: \n"
```

Comment:
Gtranslator always removes that entry :/

Reply:
Go to "Edit Header" -> "Project" and enter the URL.